### PR TITLE
Explaining how to set a custom DB connection

### PR DESCRIPTION
**Pull request by jklein**

### CODE REVIEW

File: validation.md [hunk 1]
```diff
@@ -48,6 +48,30 @@ Multiple rules may be delimited using either a "pipe" character, or as separate
...
+
+	$verifier = App::make('validation.presence');
+	$verifier->setConnection('connectionName');
+
+	$validator = Validator::make(
+        [
+            'name' => 'Dayle',
+            'password' => 'lamepassword',
+            'email' => 'email@example.com'
+        ],
```
Comment (taiyaei) [inline validation.md]: L61-65 should be tabs

**Comment by jklein** [comment]
@taiyaei changed the spaces to tabs

File: validation.md [hunk 2]
```diff
@@ -48,6 +48,30 @@ Multiple rules may be delimited using either a "pipe" character, or as separate
...
+	$verifier = App::make('validation.presence');
+	$verifier->setConnection('connectionName');
+
+	$validator = Validator::make(
+	[
+		'name' => 'Dayle',
+		'password' => 'lamepassword',
+		'email' => 'email@example.com'
+	],
+		[
```
Comment (GrahamCampbell) [inline validation.md]: The cs here is a mess. Why would you indent the second param like this?

**Comment by jklein** [comment]
@GrahamCampbell Sorry, when I changed the spaces to tabs that got screwed up. The indentation mimics the code sample above now.

File: validation.md [hunk 3]
```diff
@@ -48,6 +48,30 @@ Multiple rules may be delimited using either a "pipe" character, or as separate
...
+query the database and confirm uniqueness. To override this, do the following:
+
+	$verifier = App::make('validation.presence');
+	$verifier->setConnection('connectionName');
+
+	$validator = Validator::make(
+		[
+			'name' => 'Dayle',
+			'password' => 'lamepassword',
+			'email' => 'email@example.com'
```
Comment (GrahamCampbell) [inline validation.md]: Please include a trailing comma.

File: validation.md [hunk 4]
```diff
@@ -48,6 +48,30 @@ Multiple rules may be delimited using either a "pipe" character, or as separate
...
+		],
+		[
+			'name' => 'required',
+			'password' => 'required|min:8',
+			'email' => 'required|email|unique:users'
```
Comment (GrahamCampbell) [inline validation.md]: Please include a trailing comma.

**Comment by GrahamCampbell** [comment]
Please fix the remaining issues, then squash to one commit.

**Comment by jklein** [comment]
@GrahamCampbell There are no trailing commas in the other existing code examples. I'm happy to add them here, or add them everywhere, but if I only add them here it will be inconsistent. Let me know what you prefer.

**Comment by GrahamCampbell** [comment]
All examples should have trailing commas, it's very minor though. Not worth changing them all. They'll all get caught when we move to our new coding style guide in May.

**Comment by jklein** [comment]
That merge got messy - I'm going to close this PR and open a new one with a clean diff.

**Comment by jklein** [comment]
@GrahamCampbell see #1280

**Pull request closed without merging**